Makefile

```diff
@@ -11,8 +14,7 @@ build:
 test:
 	@mkdir -p ${CIRCLE_ARTIFACTS}
 	@go test -race -coverprofile=${CIRCLE_ARTIFACTS}/cover.out .
-	@go test -race -cover ./vendor/...
```

Comment:
I removed this line - it wasn't working with the new setup, and didn't make sense to me to test the vendored libraries anyway...

Reply:
Sounds good to me 👍